site/public/js/rainbow-customization.js

```diff
@@ -600,16 +600,25 @@ function setInputsVisibility(elem) {
 }
 
 /**
- * Sets the visibility for 'final grade cutoffs' input boxes
- * based on the 'final_grade' boxes in 'display' being selected / un-selected
+ * Sets the visibility for input boxes other than benchmark percents
+ * based on the corresponding boxes in 'display' being selected / un-selected
  * */
-function setFinalGradeCutoffsVisibility() {
-    // Sets visibility of 'final grade cutoffs' based on 'display_final_grade'
-    if ($("input[value='final_grade']:checked").val()) {
-        $('#final_grade_cutoffs').show();
+function setCustomizationItemVisibility(elem) {
+    //maps a checkbox name to the corresponding customization item id
+    const checkbox_to_cust_item = new Map([
+        ['final_grade', '#final_grade_cutoffs'],
+        ['messages', '#cust_messages'],
+        ['section', '#section_labels'],
+    ]);
```

Comment:
Would it be better to use an object for this?  Something like this:
```suggestion
    const checkbox_to_cust_item = {
        'final_grade': '#final_grade_cutoffs',
        'messages': '#cust_messages',
        'section': '#section_labels',
    };
```

Reply:
Good point. Done in [94bea58](https://github.com/Submitty/Submitty/pull/10593/commits/94bea5875106933d9e9ef2f261025d1a68b69671)